.pre-commit-config.yaml

```diff
@@ -1,8 +1,9 @@
 repos:
-  - repo: https://github.com/ambv/black
-    rev: 22.1.0
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
     hooks:
     - id: black
+      additional_dependencies: ['click==8.1.2']
```

Comment:
I don't think this line should be necessary with v0.22.3 of Black

Reply:
Good point. 
```suggestion
```